components/server/src/iam/iam-session-app.spec.ts

```diff
@@ -72,22 +101,34 @@ class TestIamSessionApp {
         await request(this.app.create())
             .post("/session")
             .set("Content-Type", "application/json")
-            .send(JSON.stringify(this.idToken));
+            .send(JSON.stringify(this.payload));
 
         expect(count, "sessions added").to.equal(1);
     }
 
-    @test public async testSessionRequestResponsesWithSetCookie() {
+    @test public async testSessionRequestResponsesWithSetCookie_createUser() {
         const result = await request(this.app.create())
             .post("/session")
             .set("Content-Type", "application/json")
-            .send(JSON.stringify(this.idToken));
+            .send(JSON.stringify(this.payload));
 
-        expect(result.statusCode).to.equal(200);
+        expect(result.statusCode, JSON.stringify(result.body)).to.equal(200);
```

Comment:
Why is the body being converted to a string, when we're only check for the status code to be 200?

Reply:
that prints potential errors if status code is not 200.